app/grandchallenge/evaluation/tasks.py

```diff
@@ -562,19 +564,80 @@ def get_average_job_duration_for_phase(phase):
     jobs = Job.objects.filter(
         outputs__evaluation_evaluations_as_input__submission__phase=phase,
     ).distinct()
+    start_date = datetime.datetime.strptime("1/1/2021", "%d/%m/%Y")
+    end_date = datetime.datetime.now()
+    delta = relativedelta.relativedelta(end_date, start_date)
+    monthly_costs = {}
+    algorithms_submitted_per_month = {}
+    for year in [start_date.year, start_date.year + delta.years]:
+        if not year == start_date.year + delta.years:
+            months = range(1, 13)
+        else:
+            months = range(1, 2 + delta.months)
+        for month in months:
```

Comment:
This is going to do a lot of queries, one per month per challenge. It should be possible with an aggregate instead, no?

Reply:
It's not clear to me yet how this would work with an aggregate, but I'm looking into it.